mappings/ace2ea0fe142a3687acf86f55bcded860a920864163ee0d3dda8b60252414b4552.json

```diff
@@ -42,7 +42,7 @@
     },
     "policy": "820182018282051a05a6272c8200581c39a1df51147b6de6689a4727846962fb6540c3a3c7859a1a79b9420f",
     "logo": {
-        "sequenceNumber": 0,
+        "sequenceNumber": 1,
```

Comment:
Hi @kristopherfana,

You need to sign the values again because once you increment the sequenceNumber, the signature for the logo also updates.

Reply:
Hey @VladislavKudrin 

How do I sign it?

Reply:
As you did before, you create a draft file -> put all needed values in -> increment sequenceNumber for a logo -> sign the file -> finalize the file.

Please refer to: 
[How to prepare entry](https://github.com/cardano-foundation/cardano-token-registry/wiki/How-to-prepare-an-entry-for-the-registry-%28NA-policy-script%29)
[How to update entry](https://github.com/cardano-foundation/cardano-token-registry/wiki/How-do-I-update-my-entry-in-the-registry%3F)

Thanks!